### PR TITLE
Close settings with escape (#4678)

### PR DESCRIPTION
**Pull request by cinqmilleans**
**Resolved / Related Issues**
Use key escape for close settings

### CODE REVIEW

File: Files/Views/Settings.xaml.cs [hunk 1]
```diff
@@ -43,6 +46,8 @@ protected override void OnNavigatedFrom(NavigationEventArgs eventArgs)
         {
             var currentView = SystemNavigationManager.GetForCurrentView();
             currentView.BackRequested -= OnBackRequested;
+
+            Window.Current.CoreWindow.Dispatcher.AcceleratorKeyActivated -= CoreDispatcher_AcceleratorKeyActivated;
```
Comment (yaichenbaum) [inline Files/Views/Settings.xaml.cs:50]: Shouldn't this be in `CoreDispatcher_AcceleratorKeyActivated`?

Comment (cinqmilleans) [reply Files/Views/Settings.xaml.cs:50]: @yair100 I used the same logic as for OnBackRequested. It unsubscribes even if we quit using the button. In your proposal, you can only unsubscribe when you exit using the keyboard shortcut.

Comment (yaichenbaum) [reply Files/Views/Settings.xaml.cs:50]: @duke7553 will this work?

File: Files/Views/Settings.xaml.cs [hunk 2]
```diff
@@ -69,6 +74,15 @@ private void GoBack()
             }
         }
 
+        private void CoreDispatcher_AcceleratorKeyActivated(CoreDispatcher sender, AcceleratorKeyEventArgs e)
+        {
+            if (!e.Handled && e.VirtualKey == VirtualKey.Escape)
```
Comment (yaichenbaum) [inline Files/Views/Settings.xaml.cs:79]: Can you change it to recognize the back key instead?

Comment (cinqmilleans) [reply Files/Views/Settings.xaml.cs:79]: Ok I do it tomorrow.

**Pull request closed without merging**